openmetadata-ui/src/main/resources/ui/src/utils/FeedUtils.tsx

```diff
@@ -200,7 +200,8 @@ export async function suggestions(
             ENTITY_URL_MAP[entityType as EntityUrlMapType],
             hit._source.name
           ),
-          type: hit._index === SearchIndex.USER ? UserTeam.User : UserTeam.Team,
+          type:
+            hit._index === SearchIndex.USER ? OwnerType.USER : OwnerType.TEAM,
           name: hit._source.name,
```

Comment:
what if its table index, it will assign it to type team?

Reply:
It will be either team or user since we are calling with their index 
<img width="541" alt="image" src="https://github.com/user-attachments/assets/b27c0727-ae37-42cb-adce-96b7aeb12742">